tests/coverage

```diff
@@ -0,0 +1 @@
+71.3
```

Comment:
I find this coverage a bit weird. I run it locally with the coverage test from rust-vmm-ci and it give me a coverage of 82.9. Maybe you're not properly excluding files?